dsl/src/main/java/tech/pegasys/peeps/node/Account.java

```diff
@@ -37,14 +37,20 @@ public enum Account {
    */
   BETA(
       "627306090abaB3A6e1400e9345bC60c78a8BEf57",
-      "8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63"),
+      "c87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3"),
   /*
    * publicKey: "fe3b557e8fb62b89f4916b721be55ceb828dbd73"
    * privateKey: "8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63"
    */
   GAMMA(
       "fe3b557e8fb62b89f4916b721be55ceb828dbd73",
-      "8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63");
+      "8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63"),
+  CHARLIE(
```

Comment:
nit: I think this is mixing greek with nato alphabet...should be alpha, beta, gamma, delta, epsilon :)

Reply:
:+1: Before you know it we will be mixing metaphors!